src/containers/Charts/Chart.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react'
+import { Bar, Line, Pie } from 'react-chartjs-2'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+import './Chart.css'
+
+
+const Chart = ({ solarData }) => {
+
+  const [chartType, updateChartType] = useState('bar')
+  const [yAxisLabel, updateYAxisLabel] = useState('(kWh / m^2) / day')
+
+  const [data, updateData] = useState({
+    labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+    datasets: [{
+      label: 'Solar Radiation',
+      backgroundColor: 'rgba(75,192,192,1)',
+      borderColor: 'whitesmoke',
+      borderWidth: 2,
+      data: solarData.solrad_monthly,
+
+    }]
+  })
+
+  const getSpecificChart = (e) => {
+    let chartName = e.target.options[e.target.selectedIndex].text
```

Comment:
Great work on this dynamic method for being able to toggle between data sets and graphs